gtsam/geometry/Cal3Bundler.h

```diff
@@ -109,6 +111,11 @@ class GTSAM_EXPORT Cal3Bundler {
     return v0_;
   }
 
+  Matrix3 K() const; ///< Standard 3*3 calibration matrix
```

Comment:
this is in base class, should be omitted or made override?

Reply:
Should override, since we should force skew to be 0, even if it is read from a file.